tests/test_async.py

```diff
@@ -350,6 +355,7 @@ def test_sync_poll(self):
         self.assertEquals(cur.fetchone()[0], 1)
 
     @slow
+    @skip_if_crdb
```

Comment:
Are the tests in this file missing a reason?

Reply:
I'll double check. I assume because `notify` is not supported

Reply:
That was the case, however the reason was missing. Added missing reasons, and fixed decorator in order to require one.